service/src/main/java/org/cbioportal/genome_nexus/service/internal/EnsemblServiceImpl.java

```diff
@@ -168,4 +169,9 @@ else if (hugoSymbols != null && hugoSymbols.size() > 0) {
             return new ArrayList<EnsemblTranscript>();
         }
     }
+
+    @Override
+    public HashMap<String, String> getHugoSymbolToEntrezGeneIdMap() {
```

Comment:
implement a function like `getEntrezGeneIdByHugoSymbol(String hugoSymbol)` instead of this one